android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapLocationListener.java

```diff
@@ -24,7 +24,7 @@
     private static final String LOG_TAG = LoggerUtil.makeLogTag(MapLocationListener.class);
     private final WeakReference<MapFragment> mMapActivity;
     private final LocationUpdateListener mNetworkLocationListener =
-            new LocationUpdateListener(LocationManager.NETWORK_PROVIDER, 1000, null);
+            new LocationUpdateListener(LocationManager.NETWORK_PROVIDER, 5000, null);
```

Comment:
Why change this? Won't it make the UI less responsive? I argue there is no evidence to power consumption decrease without significant changes to this parameters on the order of minutes. We have see how the wifi scanner is independent of request frequency (on small time differences) in the wifi scanner code in the app.

Aside:
- the second parameter is particularly ineffective for this purpose, because to detect distance changes requires the location sensor to be active. Now for GPS providers, both of these parameters have different power implications, but the type of GPS chip becomes a factor. For instance some GPS chips can use use a lower-power coarse location state to determine roughly when distance has changed, which makes the second parameter potentially useful.